nextflow.config

```diff
@@ -59,6 +59,9 @@ params {
     gm_method = "average"
     gm_delimiter = "."
 
+    // Locidex
+    batch_size = 2
```

Comment:
I would default it to something much larger (maybe 1000?). Though we can adjust this based on results of our benchmarking.

For test cases we can have two tests with smaller batch sizes (to test the `mv` condition if one batch and `csvtk` if multiple).